tasks/main.yml

```diff
@@ -109,21 +109,6 @@
   when: item.elements is defined
   no_log: true
```

Comment:
We should be able to remove no_log now - this was to avoid the output dump.

Reply:
Done